boards/arm/az3166_iotdevkit/az3166_iotdevkit_defconfig

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F4X=y
+CONFIG_SOC_STM32F412RX=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable GPIO ports
+CONFIG_GPIO=y
+
+# Enable Clocks
+CONFIG_CLOCK_CONTROL=y
+
+# enable pin controller
+CONFIG_PINCTRL=y
+
+# disable HTS221 trigger mode as the sensor's DRDY pin is not used on AZ3166
+CONFIG_HTS221_TRIGGER_NONE=y
```

Comment:
Please move this to board's Kconfig.defconfig file, as done on similar boards.
(This being said, I'm a bit confused as I don't recall the rational)

Reply:
Ah, good call!
This is actually to get rid of this kind of warning:

```

Reply:
I will also move the default setting for SSD1306_REVERSE_MODE

Reply:
Great, now I remember :-)